lib/xclarity_client/configuration.rb

```diff
@@ -3,15 +3,15 @@
 module XClarityClient
   class Configuration
 
-    attr_accessor :username, :password, :host, :csrf_token, :auth_type, :generated_token, :verify_ssl
+    attr_accessor :username, :password, :host, :csrf_token, :auth_type, :generated_token, :port, :verify_ssl
 
     def initialize(args)
 
       args.each { |key, value| send("#{key}=", value) }
 
-      if [username, password, host, verify_ssl].any? { |item| item.nil? }
-        $lxca_log.error "XClarityClient::Configuration initialize","username, password, host, and verify_ssl must all be specified"
-        raise ArgumentError, "username, password, host, and verify_ssl must all be specified"
+      if [username, password, host, port, verify_ssl].any? { |item| item.nil? }
```

Comment:
@AndreyMenezes you could check here if the port is valid. 
I think it can considered valid if it was greater than 0, what do you think @rodneyhbrown7?